tests/test_transfer_products.py

```diff
@@ -53,19 +53,27 @@
 EXPECTED_OBJECTS_TO_COPY = [
```

Comment:
The `transfer_products()` function inspects the files in the HyP3 bucket, compares to the files in the PDC bucket, and acts on files that haven't already been transferred.

We still want to test that files that already exist at the target prefix are *not* copied. You'll want to update `EXISTING_OBJECTS` to match the new prefix format, and test that `EXPECTED_OBJECTS_TO_COPY` only has four items, not six.